cloudinit/userdata_cloudinit.erb

```diff
@@ -20,6 +20,13 @@ oses:
 hostname: <%= @host.shortname %>
 fqdn: <%= @host %>
 manage_etc_hosts: true
+ssh_pwauth: <%= @host.params['ssh_pwauth'] || true %>
+<% if @host.params['ssh_authorized_keys'] -%>
```

Comment:
There are quite some places where variables are used and not checked against `''`, so IMHO we can merge this and have a general clean-up on the whole tree later.